registrations/admin.py

```diff
@@ -65,13 +104,24 @@ class RegistrationAdmin(RegistrationBaseAdmin, TranslationAdmin, VersionAdmin):
     )
     list_filter = (EventFilter,)
     autocomplete_fields = ("event",)
-    inlines = (RegistrationUserAccessInline, RegistrationPriceGroupInline)
+    inlines = (
+        RegistrationUserAccessInline,
+        RegistrationPriceGroupInline,
+        RegistrationWebStoreMerchantInline,
+        RegistrationWebStoreAccountInline,
+    )
 
     def get_formsets_with_inlines(self, request, obj=None):
         for inline in self.get_inline_instances(request, obj):
-            # hide RegistrationPriceGroupInline if Talpa integration is not enabled
+            # hide Talpa-related inline forms if Talpa integration is not enabled
             if (
-                not isinstance(inline, RegistrationPriceGroupInline)
+                not any(
+                    [
+                        isinstance(inline, RegistrationPriceGroupInline),
+                        isinstance(inline, RegistrationWebStoreMerchantInline),
+                        isinstance(inline, RegistrationWebStoreAccountInline),
+                    ]
+                )
                 or settings.WEB_STORE_INTEGRATION_ENABLED
```

Comment:
Not really an issue, but personally I'd probably check `settings.WEB_STORE_INTEGRATION_ENABLED` first since it's probably a less complex check to make and since we're doing `or` we most likely don't need to do the more complex check. :shrug: